### DISCUSSION TITLE
PDF styling options for figures with class "is-pulled-right"

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#955] [Get Help]
When figures with the class "is-pulled-right" appear in a pdf, they are full-column width and appear before the paragraph in which they are mentioned, since the figure shortcode has to be placed in the markdown above the paragraph in which you would like the figure pulled right in the web version. This can create some awkwardness in the pdf (for example, if the image appears immediately after a header), and doesn't allow the use of float options such as "pdf-float-top." Would it be possible to create an alternative placement or have alternative pdf classes for these images? If not, it would be helpful if these images could by default end up _after_ the paragraph in the pdf, so they could appear closer to other images that appear after the paragraph.

**Comment by Erin-Cecele** [comment]
Hi @zsofiaj, was this ever resolved, or are you still waiting on a solution from us?

**Comment by zsofiaj** [comment]
Thanks so much for following, @Erin-Cecele . I was still hoping you might have a solution for this. Is there a way to control where the "is-pulled-right" images land in the pdf? I've been trying to avoid using that float option, since it limits how much I can control the layout of the pdf, but there are a handful of places in our project where we wanted the images smaller and inset within the paragraph.

**Comment by Erin-Cecele** [comment]
@jenpark-getty or @geealbers Do you have any potential solutions for Zsofia?
